node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
That should be handled via a combination of semantic-versioning labels and npm tags: standard practice for managing prerelease npm packages.

---
In reply to: [73274972](https://github.com/openT2T/opent2t/pull/2#discussion_r73274972) [](ancestors = 73274972)